package.json

```diff
@@ -2,9 +2,11 @@
   "name": "node-elm-compiler",
   "version": "5.0.1",
   "description": "A Node.js interface to the Elm compiler binaries.",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
Should this be `dist/index.js` now? I assume we want the published code to use the compiled `index.js`, yeah?

Reply:
Is there a way to add a test for this?

I could imagine modifying the tests to run on the compiled output by importing the root directory containing the `package.json`. I think that Node correctly resolves the main file in that case.

I did not do that yet, because I thought that the compilation step would make it less convenient to test during development. But I guess the compilation step has to happen anyway and is right now "hidden" in `ts-mocha`. So maybe it's worth making it explicit?

Reply:
I've created a PR for this on my fork: https://github.com/Y0hy0h/node-elm-compiler/pull/2